app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None | str:
+    if command.startswith("mv "):
+        pass
```

Comment:
what if the command is 'mvp'

Reply:
The pass statement is used as a placeholder when you need something syntactically but don't want it to do anything

Reply:
> what if the command is 'mvp'

but I check if it starts with "mv " not "mv"